filament/src/components/TransformManager.h

```diff
@@ -189,11 +189,11 @@ class UTILS_PRIVATE FTransformManager : public TransformManager {
         WORLD,          // world transform
         LOCAL_LO,       // accurate local translation
         WORLD_LO,       // accurate world translation
-        MATERIAL_LOCAL_ORIENTATION,         // local orientation of bi/triplanar mapping
-        MATERIAL_ORIENTATION,               // bi/triplanar mapping rotation matrix, composed with parent transforms
+        MATERIAL_LOCAL_ORIENTATION,    // local orientation quaternion of bi/triplanar mapping
+        MATERIAL_ORIENTATION,          // bi/triplanar mapping rotation quaternion, composed with parent transforms
```

Comment:
nit: lets not change the indentation
```suggestion
        MATERIAL_LOCAL_ORIENTATION,         // local orientation quaternion of bi/triplanar mapping
        MATERIAL_ORIENTATION,               // bi/triplanar mapping rotation quaternion, composed with parent transforms
```

Reply:
https://github.com/shapr3d/filament/pull/162/commits/6b05fb0008899ce758a2cd8cfc8e1e4b2f094c46